wrapper/src/main/java/software/amazon/jdbc/PluginServiceImpl.java

```diff
@@ -248,7 +250,7 @@ protected EnumSet<NodeChangeOptions> compare(
     if (hostSpecA.getAvailability() != hostSpecB.getAvailability()) {
       if (hostSpecB.getAvailability() == HostAvailability.AVAILABLE) {
         changes.add(NodeChangeOptions.WENT_UP);
-      } else if (hostSpecB.getAvailability() == HostAvailability.NOT_AVAILABLE) {
+      } else if (hostSpecB.getAvailability() == HostAvailability.UNAVAILABLE) {
```

Comment:
Renaming this enum value, is it a breaking change?

Reply:
AH Good call out. I'll change it back.

Reply:
Done!